cob_default_env_config/ipa-apartment/polygon_maps/restriction_polygon.yaml

```diff
@@ -0,0 +1,4 @@
+restricted areas: 
+  - id: 31287680
```

Comment:
so where in the apartment is this?
do we have a visualization, e.g. screenshot from rviz?